src/Chirp.Web/Pages/Auth.cshtml.cs

```diff
@@ -24,8 +26,10 @@ public ActionResult OnGet(bool? signOut)
         var userEmail = User?.FindFirst(ClaimTypes.Email)?.Value;
         var userId = int.Parse(User?.FindFirst(ClaimTypes.NameIdentifier)?.Value ?? "0");
 
+        _userData.Author = _authorRepository.FindAuthorsByName(userName ?? "").FirstOrDefault();
```

Comment:
should be placed under _authorRepository.CreateAuthor(...) if hasn't been created yet